### ISSUE TITLE
add softmax activation function and regression

### ISSUE THREAD

**Issue by emirkmo** [emirkmo/andrew-ng-ML-course-labs#8]
For multi-class regression and using in neural nets.
